incremental/CavernousII/src/script/loop_log.ts

```diff
@@ -44,6 +44,12 @@ class LoopLog {
 	}
 
 	finalize(){
+		this.stats = stats.map(s => {
+			return {
+				base: s.base,
+				current: s.base,
+			};
+		});
```

Comment:
Not sure how this is intended to work; it prevents any increase of base being saved in the loop log.